frontend/src/component/ui/ChangeLogModal.spec.tsx

```diff
@@ -15,19 +15,25 @@ describe('ChangeLogModal', () => {
     const changeLogModalRef = React.createRef<ChangeLogModal>();
     const changeLog = [
       {
+        id: '12345',
         title: 'Lorem ipsum',
-        releasedAt: 1500000000003,
-        summary: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit'
+        releasedAt: new Date(1500000000003),
```

Comment:
I recommend doing https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse so that the dates are readable.